sf-perftest-testapp/locust/README.md

```diff
@@ -0,0 +1,23 @@
+# Locust experiment
+
+This experiment utilizes [Locust](https://locust.io/) to drive traffic against the Test App.
+This experiment is a demonstration of the [Locust trial image](https://github.com/thestormforge/optimize-trials/tree/main/locust) provided by StormForge to enable the use of Locust in Optimize Pro trial jobs.
+
+## Prerequisites
+
+- A Kubernetes cluster
+- `kubectl` configured to connect to your cluster
+- [Optimize Pro](https://docs.stormforge.io/optimize-pro/getting-started/install/) installed on your cluster
+- The StormForge PerfTest [Test App](../application/README.md) deployed on your cluster
+
+## Running the Experiment
+
+- Edit the `name` field in the `experiment.yaml` file, setting it to something unique
+- Inject the JMeter test plan file into a configmap in your cluster with the following command:
+  ```sh
+  kubectl create configmap locustconfig --from-file=./locustfile.py
```

Comment:
If we ask users to create a configmap here, we don't need it in the `experiment.yaml`, do we?

Reply:
Good point. Removed in https://github.com/thestormforge/examples/pull/75/commits/925c964b81087f8ee111f58323980f55bf287bec.